glossary-entries.tex

```diff
@@ -54,3 +51,26 @@
 \newacronym{hdd}{HDD}{hard disk drive}
 \newacronym{ecc}{ECC}{error correcting code}
 \newacronym{ascii}{ASCII}{American Standard Code for Information Interchange}
+\newacronym{api}{API}{Application Programming Interface}
```

Comment:
In a previous PR we've decided to change this to lowercase. Please outline why this change is required - it might look weird in other texts now.

Reply:
So what about all other acronyms? Should everything be lowercase?
I for one think it looks terrible.

Reply:
The correct way to do this is to have the acronym's first usage in the text as lowercase, and in the glossary as uppercase (except for names of course).
Therefore, please change your added entries to be all lowercase. We'll make the first character appear uppercase in another PR.

Reply:
The correct way to do this would be to have the acronym's first usage in the text as lowercase, and in the glossary as uppercase (except for names of course).
Therefore, please change your added entries to be all lowercase. We'll make the first character appear uppercase in another PR.

Reply:
I did it but can you please cite a reference which says that this is the correct way?

Reply:
This was discussed with @danielrutz - we both agreed on this. Unfortunately, I can not provide you a reference that says so.